lib/team_manager.rb

```diff
@@ -27,15 +31,11 @@ def game_info(game_id)
   end
 
   def gather_game_team_info(id)
-    game_ids_by_team(id).map do |game_id|
-      game_team_info(game_id)
-    end
+    stat_tracker.gather_game_team_info(game_ids_by_team(id))
   end
```

Comment:
Very nice refactoring. Moving the logic elsewhere is really nice for organization.